packages/frontend-2/pages/server-management/active-users.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink
+        :to="'/server-management'"
```

Comment:
If the value to a prop is a string literal, you can remove the `:` to make Vue not treat the value as JS code and just do `to="/server-management"`

Same goes for the other link